src/inject/settings/hotkeys.js

```diff
@@ -9,13 +9,36 @@ const MODIFIER_KEYS = {
   18: 'Alt',
 };
 
-const ACTION_KEYS = _.transform(_.range(26), (final, current) => {
+const ACTION_KEYS = _.transform(_.range(26), (final, current) => { // letters
   final[current + 65] = String.fromCharCode(current + 65); // eslint-disable-line
+  if (current < 12) {
```

Comment:
That was how it was written in the original commit, but I thought this format was decidedly more readable. I can change it back, though.